weblate/trans/models/unit.py

```diff
@@ -981,6 +983,7 @@ def update_from_unit(  # noqa: C901,PLR0914
         unit = unit_attributes["unit"]
         created = unit_attributes["created"]
         pos = unit_attributes["pos"]
+        automatically_translated = unit_attributes["automatically_translated"]
```

Comment:
Wondering if this field should be checked as well in the `same_metadata` check now? I don't think most formats support writing the `automatically_translated` flag so we'll need the same treatment as `is_fuzzy` at least if we do.

(https://github.com/WeblateOrg/weblate/blob/b85f8d2c35aad35fa29bba11cb2ff4b275672498/weblate/trans/models/unit.py#L1059)